src/index.js

```diff
@@ -1,23 +1,36 @@
 import './index.css';
+import TodoList, {
+  render,
+  inputTodo,
+  editDescription,
+} from './module/AddRemove.js';
+import './module/editableContent';
 
-const todoListContianer = document.querySelector('.todo__list');
+const TodoListObj = new TodoList();
+function Starter() {
+  const threeDots = document.querySelectorAll('li');
+  threeDots.forEach((dotValue, index) => {
+    const dot = dotValue.querySelector('.three__dots');
+    if (dot) {
+      const newDot = dot.cloneNode(true);
+      dotValue.replaceChild(newDot, dot);
+      newDot.addEventListener('click', () => {
+        editDescription(dotValue, index, TodoListObj);
+        Starter();
+      });
+    }
+  });
+}
 
-// const li = document.createElement('li');
+inputTodo.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    TodoListObj.addTodo();
+    render(TodoListObj.list, TodoListObj);
+    Starter();
+  }
+});
 
-const listTodo = [
-  { description: 'wash the dishes', completed: false, index: 0 },
-  { description: 'Complete To Do list project', completed: false, index: 1 },
-  { description: 'Fix Car', completed: false, index: 2 },
-];
+// Create
 
-listTodo.forEach((list) => {
-  todoListContianer.innerHTML += `<li>
-  <div class="check">
-    <input type="checkbox" name="completed" />
-    <p>${list.description}</p>
```

Comment:
- [ ] I can see that the check box does not hardcoded items or book, The name of  books are supposed to be 

Original:

![edittttt](https://user-images.githubusercontent.com/57562869/149338316-d0433cc6-8991-4b85-a218-4af718a5526a.png)

Yours:

Reply:
The delete is done on the hover of the three dots to prompt a delete. the next feature is the clear completed with is on interactivity. using the event listener on checkbox on Change

Reply:
HI Ginohmk, sorry for the misunderstanding, I will do this as an option for the second review to approve your project

Reply:
Thank you. I really appreciate it. And thanks for your reply